route_guide/route_guide_server.cc

```diff
@@ -55,133 +64,235 @@ float GetDistance(const Point& start, const Point& end) {
   float lon_2 = end.longitude() / kCoordFactor;
   float lat_rad_1 = ConvertToRadians(lat_1);
   float lat_rad_2 = ConvertToRadians(lat_2);
-  float delta_lat_rad = ConvertToRadians(lat_2-lat_1);
-  float delta_lon_rad = ConvertToRadians(lon_2-lon_1);
+  float delta_lat_rad = ConvertToRadians(lat_2 - lat_1);
+  float delta_lon_rad = ConvertToRadians(lon_2 - lon_1);
 
-  float a = pow(sin(delta_lat_rad/2), 2) + cos(lat_rad_1) * cos(lat_rad_2) *
-            pow(sin(delta_lon_rad/2), 2);
-  float c = 2 * atan2(sqrt(a), sqrt(1-a));
+  float a = pow(sin(delta_lat_rad / 2), 2)
+      + cos(lat_rad_1) * cos(lat_rad_2) * pow(sin(delta_lon_rad / 2), 2);
+  float c = 2 * atan2(sqrt(a), sqrt(1 - a));
   int R = 6371000; // metres
 
   return R * c;
 }
 
-std::string GetFeatureName(const Point& point,
-                           const std::vector<Feature>& feature_list) {
+// Just return the name of feature in feature_list vector
```

Comment:
It's nicer to skip words like "just" in comments. For two reasons:
1. Whether something is "just X" is a matter of opinion. It may be obvious to you, but no need to make anyone else feel bad if they don't "just" get it. 😉 
2. It doesn't convey any important information, which means it's distracting from the important message.

So "just" omit it! 😉 For bonus points you can also be clearer about which feature we're talking about:
```suggestion
// Return the name of the feature in the `feature_list` vector that's associated with the given `point`.
```